dot/sync/test_helpers.go

```diff
@@ -88,3 +87,5 @@ func BuildBlock(t *testing.T, instance runtime.Instance, parent *types.Header, e
 		Body:   body,
 	}
 }
+
+const defaultSlotDuration = time.Second * 6
```

Comment:
supernit moving the unit after the multiplier!

```suggestion
const defaultSlotDuration = 6 * time.Second
```

Reply:
Updated, Question: Why?  Is this for readability or is there some other reason?

Reply:
It's just me being [pedantic](https://english.stackexchange.com/questions/2794/punctuation-with-units#:~:text=From%20the%20SI%20Brochure%2C%20%C2%A7,the%20unit%20from%20the%20number.), the unit should be after the numerical value. Like `2 Lbs`, not `Lbs 2` 😉